src/ucp/core/ucp_worker.c

```diff
@@ -1462,6 +1462,17 @@ unsigned ucp_worker_get_ep_config(ucp_worker_h worker,
     return config_idx;
 }
 
+static void ucp_worker_warn_config_unused_env_vars()
```

Comment:
can we move this to ucs? so gtest will call internal function to do the testing and make sure warning is printed, but ucp_worker_create will still be calling ucs_config_parser_warn_unused_env_vars() like before

Reply:
sure, we can do it - done